resolwe/test/testcases.py

```diff
@@ -146,29 +147,31 @@ def setUp(self):
                                           FLOW_DOCKER_MAPPINGS=flow_docker_mappings)
         self.settings.enable()
 
-        self._keep_all = False
-        self._keep_failed = False
+        self._keep_data = False
 
         user_model = get_user_model()
         self.admin = user_model.objects.create_superuser(username='admin', email='admin@test.com', password='admin')
         self.contributor = user_model.objects.create_user(username='contributor')
 
     def tearDown(self):
         """Clean up after the test."""
-        if not self._keep_all and not self._keep_failed:
+        if not self._keep_data:
             shutil.rmtree(settings.FLOW_EXECUTOR['DATA_DIR'], ignore_errors=True)
 
         self.settings.disable()
 
         super(TransactionTestCase, self).tearDown()
 
-    def keep_all(self):
-        """Do not delete output files after test for all data."""
-        self._keep_all = True
+    def keep_data(self, mock_purge=True):
```

Comment:
I don't like `mock_purge` name. User who will use this doesn't need to know what mock is. It should describe what it does, not how it does.